load_balance.go

```diff
@@ -339,9 +339,9 @@ func refreshLoadInfo(li *ClusterLoadInfo) error {
 	var host, nodeType, cloud, region, zone, publicIP string
 	var port, numConns int
 	newHostLoad := make(map[string]int)
-	li.hostPort = make(map[string]uint16)
-	li.zoneList = make(map[string][]string)
-	li.hostPairs = make(map[string]string)
+	newHostPort := make(map[string]uint16)
```

Comment:
Why is this change needed?

Reply:
Discussed in call, I have also mentioned the reason in the [doc](https://docs.google.com/document/d/15cTZ--6OZaYr7_Dxs9vXK_j2Uppn3OkQh7eKWRz0Bn8/edit#heading=h.f27v1pysns58) that is shared above.